dashboard/components/Widgets.tsx

```diff
@@ -9,58 +24,216 @@ const enum WidgetHeight {
   Small = 216,
 }
 
-function lazyLoadWidget(
-  importPromise: () => LoaderComponent,
-  loaderSize?: number
-) {
-  return dynamic(importPromise, {
-    loading: () => (
-      <Widget>
-        <Widget.Content status="loading" loaderSize={loaderSize} />
-      </Widget>
-    ),
-    ssr: false,
-  })
-}
+export default function Widgets() {
+  const { query } = useRouter()
+  const { host, token } = getConfig(
+    typeof query === 'string' ? query : undefined
+  )
+  function buildEndointUrl(host: string, endpoint: string) {
+    const apiUrl =
+      {
+        'https://ui.tinybird.co': 'https://api.tinybird.co',
+        'https://ui.us-east.tinybird.co': 'https://api.us-east.tinybird.co',
+      }[host] ?? host
```

Comment:
I saw that the `host` is defined by an env var or the url host. Is this something that needs to be done to maintain some sort of backwards compatibility?

Reply:
This is the old way of defining the api host. We can fix this in another PR.